language/move-vm/runtime/src/loader.rs

```diff
@@ -1001,10 +1001,14 @@ impl Function {
         let handle = module.function_handle_at(def.function);
         let name = module.identifier_at(handle.name).to_owned();
         let scope = Scope::Module(module.self_id());
-        let code = def.code.code.clone();
+        // TODO: native functions do not have a code unit. Revisit.
```

Comment:
I would remove the TODO and just leave 
`// native functions do not have a code unit`
or nothing, up to you. In 3662 we load the native function here and we can review the structure of `Function` at some point and as needed but it should be all good